avocado/core/safeloader/core.py

```diff
@@ -318,6 +303,21 @@ def find_python_tests(module_name, class_name, determine_match, path):
     return result, disabled
 
 
+def _determine_match_avocado(module, klass, docstring):
```

Comment:
One suggestion that I'm trying to follow is to order them alphabetically.

Reply:
@beraldoleal yeah, makes sense. I'd love to have something like isort that helps with the ordering of classes/methods too.  Let's shop around to see if there's any tool that can help with that.